tests/vm_test.go

```diff
@@ -978,6 +979,33 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			})
 		})
 	})
+
+	Context("with oc/kubectl", func() {
+		var vm *v1.VirtualMachine
+		var err error
+		var vmJson string
+
+		// Get proper commandline client: oc or kubectl
+		k8sClient := tests.GetK8sCmdClient()
+
+		AfterEach(func() {
+			if vmJson != "" {
+				err = os.Remove(vmJson)
+				Expect(err).ToNot(HaveOccurred())
+				vmJson = ""
+			}
+		})
+
+		It("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]should fail when deleting non existing VM", func() {
```

Comment:
Could you please change it to:
		It("[test_id:233][posneg:negative]should fail when deleting non existing VM", func() {
￼